css/css-multicol/non-adjacent-spanners-001.html

```diff
@@ -3,7 +3,7 @@
 <link rel="help" href="https://www.w3.org/TR/css-multicol-1/#spanning-columns">
 <link rel="match" href="../reference/ref-filled-green-100px-square.xht">
 <p>Test passes if there is a filled green square and <strong>no red</strong>.</p>
-<div style="columns:2; column-fill:auto; column-gap:0; width:100px; background:red;">
+<div style="columns:2; column-fill:column; column-gap:0; width:100px; background:red;">
```

Comment:
You probably mean column-fill:balance. :) But we can also just drop the declaration, since balance is the initial value. I actually think this declaration was added by mistake, but in Blink it doesn't make any difference. It would be wrong to consume the 200px before the spanner, and I'd like to point out that Blink doesn't do that. The difference between Gecko and Blink is that Blink forcefully balances when height is unspecified (which is what the spec used to say that we should).

Reply:
Ah, yes, I meant to use `column-fill:balance`. I'll remove `column-fill:auto;` in my next revision.

> The difference between Gecko and Blink is that Blink forcefully balances when height is unspecified (which is what the spec used to say that we should).

This is what I initially thought. I made the assumption by testing this on Google Chrome 86 with `border: 1px solid blue` added to the spanner. I see the blue spanner at the bottom edge of the green rectangle. If I add an arbitrary element after the element like `<div>abc</div`, the blue spanner is at the top edge of the green rectangle, which is what I expected. Anyway, I'll modify the extended commit message to avoid misleading information about blink's behavior.

Reply:
Indeed. Something looks really wrong in the legacy implementation (which is the one shipping). I was referring to the WIP LayoutNG block fragmentation implementation (which was far from obvious in my comment - sorry).

Reply:
Is there a flag in `chrome://flags/` that can turn on LayoutNG for block fragmentation? I've searched "layout" in `chrome://flags/` but not spotted something related.

Reply:
Nothing in chrome://flags/ , at least not yet. You can enable it via the command line argument --enable-blink-features=LayoutNGBlockFragmentation . Be sure to use Chrome Canary or Dev, to get the latest stuff. :)

Reply:
The command-line argument works for me. Thanks!